pages/common/flask.md

```diff
@@ -0,0 +1,16 @@
+# flask
+
+> A general utility script for Flask applications. Loads the application defined in the $FLASK_APP environment variable.
+> More information: <https://flask.palletsprojects.com/en/1.1.x/cli/>.
+
+- Run a development server:
+
+`flask run`
+
+- Show the routes for the app:
+
+`flask routes`
+
+- Run a shell in the app context:
```

Comment:
Is this a bash shell etc, or a Python REPL?

Reply:
Second one, maybe it should be specified? Like:

    - Run a Python interactive shell in the app's context:

Reply:
added @mebeim 's suggestion